src/test/java/org/mdbenefits/app/preparers/HouseholdExpensesPreparerTest.java

```diff
@@ -46,4 +46,11 @@ public void shouldAddFieldsForHouseholdExpensesFrequency() {
         assertThat(result.get("homeExpensePhoneFrequency"))
             .isEqualTo(new SingleField("homeExpensePhoneFrequency", "Monthly", null));
     }
+
+    private Map<String, SubmissionField> withNoMedicalExpenses(Map<String, SubmissionField> results) {
+        assertThat(results.keySet()).contains("hasMedicalExpenses");
```

Comment:
This method is doing double duty - it's removing medical expenses and asserting that they are there and then gone.  

I think it should only do one thing -- remove medical expenses if they are there.  This assumes they are there and asserts if not.  

I would separate these concerns - if this method is meant to strip out medical expenses, if there, then have it just do that and not assert in the process. I think the assert in a method that hands something back to later assert seems way too complicated to me.